cmd/sidecars/README.md

```diff
@@ -180,16 +182,22 @@ the annotations:
 ```yaml
 annotations:
   hooks.kubevirt.io/hookSidecars: '[{"args": ["--version", "v1alpha2"],
-    "image": "registry:5000/kubevirt/sidecar-shim:devel",
     "configMap": {"name": "my-config-map", "key": "my_script.sh", "hookPath": "/usr/bin/onDefineDomain"}}]'
 ```
 
+Please notice that annotations set on VMs are not automatically propagated to VMIs so,
+in the case of a VM, the VM owner should configure it on `/spec/template/metadata/annotations`
+instead of directly annotating the VM as in [this example](../../examples/vm-cirros-with-sidecar-hook-configmap.yaml).
```

Comment:
Yes, you are correct with the distinction between configuring VMs, thanks for noticing!